src/ch01/reportPrimes.test.ts

```diff
@@ -14,10 +14,12 @@ const isPrime = (num: number) => {
 
 const reportPrimes = (n: number) => {
   for (let i = 2; i < n; i++) {
-    if (isPrime(i)) {
-      console.log(`${i} is prime`);
-    }
+    if (isPrime(i)) reportIfPrime(i);
   }
 };
 
+const reportIfPrime = (i: number) => {
+  console.log(`${i} is prime`);
+}
+
```

Comment:
```suggestion
const reportIfPrime = (i: number) => {
  console.log(`${i} is prime`);
}

```
화살표 함수를 사용 하셨는데 화살표 함수의 특징은 한 줄로 표현 할 수 있다는 장점을 가지고 있습니다. 현재 함수에서는 그 장점을 활용하지 않아보입니다.

Reply:
말씀대로 원본 코드를 수정했습니다